src/util/baseElement.js

```diff
@@ -4,14 +4,19 @@
  * override where you can specify any element / component as
  * the root
  */
-const baseElement = ({ href, as }) => {
+const baseElement = ({ href, onClick, as }) => {
   let Element;
   if (as) {
     Element = as;
+  } else if (href && onClick) {
+    // This may seem unnecessary but needs to be here to enable the right priority
```

Comment:
`href` isn't a valid `button` property - should `baseElement` be using an anchor when `href` is passed?